compute_tools/src/compute.rs

```diff
@@ -442,7 +442,7 @@ impl ComputeNode {
 
         let pg = self.start_postgres(spec.storage_auth_token.clone())?;
 
-        if spec.spec.mode == ComputeMode::Primary {
+        if spec.spec.mode == ComputeMode::Primary && !spec.spec.already_configured {
```

Comment:
what happened when customers create roles via control plane? If I'm not mistaken it sends the apply_config, isn't it?

Reply:
It does, yes, but the idea is that in this case control-plane will just set `already_configured: false`